presto-docs/src/main/sphinx/release/release-0.234.rst

```diff
@@ -0,0 +1,48 @@
+Release 0.234
+=============
+
+General Changes
+_______________
+* Fix query failure for cross-joining bucketed tables with ``COALESCE`` when the bucket counts are compatible but mismatched.
+* Fix query failure for window functions with ``IGNORE NULLS``.
+* Fix a regression in the new scheduler temporarily by setting the default for the configuration property ``use-legacy-scheduler`` to ``true``.
+* Improve system reliability by preventing additional queries to execute when the total number of tasks exceeds the limit, which can be
+  set by configuration property ``experimental.max-total-running-task-count-to-not-execute-new-query``.
+* Improve the scale writer heuristics by considering overall producer buffer utilization. This can be enabled by using the session property
+  ``optimized_scale_writer_producer_buffer`` and the configuration property ``optimized-scale-writer-producer-buffer``.
+* Improve connection pooling to avoid running out of sockets.
+* Add :ref:`khyperloglog_type` type and related functions.
+* Add support for streaming mark distinct changes even when materialized exchange is enabled via the
+  ``query.use-streaming-exchange-for-mark-distinct`` property. Mark distinct is enabled by default and it introduces a
+  stage per distinct column in a query in order to redistribute the tracking of distinct values across the entire cluster.
+  With materialized exchange the cost of these additional exchanges is higher. Enabling streaming exchange if there is
+  sufficient memory available to perform the distincting will reduce the cost of these additional stages while still
+  reducing the memory used by other by non-Mark Distinct stages.
+* Add support for end point ``/v1/resourceGroupState`` to return resource groups info with more filters.
+  It is possible to filter the dynamically generated resource groups.
+
+Geospatial Changes
+__________________
+* Add support for spatial joins for join condition ``ST_Distance(p1, p2) < r``.
+* Improve storage efficiency for Bing tiles by storing Bing tiles as ``BIGINT``. This also reduces bucket skew in certain cases.
+
+Hive Changes
+____________
+* Add ``ZSTD`` support for writing ``ORC`` and ``DWRF`` files. This can be enabled by setting session property ``hive.compression_codec=ZSTD``.
+
+Verifier Changes
+________________
+* Add support for verifying ``SELECT`` queries that produce ``DATE`` or ``UNKNOWN`` (null) columns, or structured typed columns with ``DATE`` or ``UNKNOWN``.
+* Add support for auto-resolving control check query failures due to ``EXCEEDED_TIME_LIMIT``.
+* Add determinism analysis support for simple queries with top-level ``ORDER BY LIMIT`` clause. (:pr:`14181`).
+
+SPI Changes
+___________
+* Add parameter ``AccessControlContext context`` to all methods in ``SystemAccessControl``.
+* Add ``firstDynamicSegmentPosition`` to SelectionContext.
+
+Druid Changes
+_____________
+* Add support for aggregation pushdown.
+* Add support for ``LIMIT`` evaluation pushdown.
```

Comment:
just merged another two druid commits:
* count distinct pushdown for druid connector
* fix druid connector segment scan

Reply:
https://github.com/prestodb/presto/commit/1e51f2f2d52b1f7ce0719330eea597bb274d077b
https://github.com/prestodb/presto/commit/5f89b532e12ef653ad2a5acc577bd5acdf5f8eb6

Reply:
Those changes will be 0.235 but not in 0.234.